### DISCUSSION TITLE
Flutter Polls v0.0.9

### DISCUSSION THREAD

**Discussion by thenifemi** [thenifemi/flutter_polls#13] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/thenifemi/flutter_polls/releases/tag/'>Flutter Polls v0.0.9</a>.</em>

**Comment by herrchmafi** [comment]
Poll options (that haven't been voted on yet) show a border by default. I thought supplying a null to `pollOptionsBorder` would remove this, but this isn't the case. Can you provide support to remove or is there something I'm missing in the API? Thank you!

**Reply by thenifemi** [reply]
Hey @herrchmafi, I apologise for the late response. Here you go - 
`                pollOptionsBorder: Border.all(style: BorderStyle.none),
`
Let me know if that helps.
